cpp/oneapi/dal/algo/knn/common.hpp

```diff
@@ -39,6 +40,9 @@ namespace v1 {
 /// Tag-type that parameterizes entities used for solving
 /// :capterm:`classification problem <classification>`.
 struct classification {};
+
+/// Tag-type that parameterizes entities used for solving
+/// the :capterm:`search problem <search>`.
```

Comment:
This line produces a warning: ` WARNING: term not in glossary: Search`. Have you defined "Search" as a term?

Reply:
Yes. It was defined